WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift

```diff
@@ -140,9 +140,18 @@ class NotificationSettingDetailsViewController: UITableViewController {
 
         // Single Section Mode: A single section will contain all of the rows
         if singleSectionMode {
-            return [Section(rows: rows)]
+            // Switch on stream type to provide descriptive text in footer for more context
+            switch stream.kind {
+            case .Device:
+                return [Section(rows: rows, footerText: NSLocalizedString("Settings for push notifications that appear on your mobile device.", comment: "Descriptive text for the Push Notifications Settings"))]
+            case .Email:
+                return [Section(rows: rows, footerText: NSLocalizedString("Settings for notifications that are sent to the email tied to your account.", comment: "Descriptive text for the Email Notifications Settings"))]
+            case .Timeline:
+                return [Section(rows: rows, footerText: NSLocalizedString("Settings for notifications that appear in the Notifications tab.", comment: "Descriptive text for the Notifications Tab Settings"))]
+            }
         }
 
+
```

Comment:
These changes are already in `develop` (I see they're from a previous PR of yours), any idea why they show up here? A stale branch perhaps?

Reply:
Also, I couldn't run this in Xcode 12 because it doesn't include the latest from `develop`, which fixes a few compile-time errors that Xcode 12 throws.

Reply:
I think it must be because my branch wasn't up to date enough, then it's trying to re-push those changes. Whoops!

Reply:
Feel free to re-request a review once your branch is up-to-date 👍